JSIL/ILBlockTranslator.cs

```diff
@@ -95,6 +97,20 @@ public class ILBlockTranslator {
 
             SpecialIdentifiers = translator.GetSpecialIdentifiers(TypeSystem);
 
+            _rawTypes = new HashSet<TypeReference>
+            {
+                TypeSystem.Boolean,
+                TypeSystem.SByte,
+                TypeSystem.Byte,
+                TypeSystem.Int16,
+                TypeSystem.UInt16,
+                TypeSystem.Int32,
+                TypeSystem.UInt32,
+                TypeSystem.Single,
+                TypeSystem.Double,
+                TypeSystem.Char
+            };
```

Comment:
I've seen, that int64 could use raw type also based on configuration. We probably need check it here?